apps/kindergarten/views.py

```diff
@@ -1 +1,10 @@
-# Create your views here.
+from django.views.generic import ListView
+
+
+from apps.kindergarten.models import Kindergarten
+
+
+class Kindergartenview(ListView):
+    model= Kindergarten
```

Comment:
you can add 
queryset = Kindergarten.objects.select_related('district').all()
to optimize db request